src/sentry/static/sentry/app/views/eventsV2/resultsChart.tsx

```diff
@@ -2,48 +2,127 @@ import React from 'react';
 import styled from '@emotion/styled';
 import * as ReactRouter from 'react-router';
 import {Location} from 'history';
+import isEqual from 'lodash/isEqual';
 
 import {Organization} from 'app/types';
-
+import {getUtcToLocalDateObject} from 'app/utils/dates';
+import {Client} from 'app/api';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
 import {Panel} from 'app/components/panels';
 import getDynamicText from 'app/utils/getDynamicText';
-import EventsChart from 'app/views/events/eventsChart';
+import {EventsChart} from 'app/views/events/eventsChart';
 
 import ChartFooter from './chartFooter';
 import EventView from './eventView';
 
-type Props = {
+type ResultsChartProps = {
+  api: Client;
   router: ReactRouter.InjectedRouter;
   organization: Organization;
   eventView: EventView;
   location: Location;
-
-  total: number | null;
-  onAxisChange: (value: string) => void;
 };
 
-export default class ResultsChart extends React.Component<Props> {
+class ResultsChart extends React.Component<ResultsChartProps> {
+  shouldComponentUpdate(nextProps: ResultsChartProps) {
+    const {eventView, ...restProps} = this.props;
+    const {eventView: nextEventView, ...restNextProps} = nextProps;
+
+    if (!eventView.isEqualTo(nextEventView)) {
```

Comment:
Does this include the UTC option from global selection?

Reply:
It doesn't. But the `isEquals()` performed after this if statement covers that since it resides in the `location` object.